packages/itwinui-react/src/core/Surface/Surface.tsx

```diff
@@ -29,7 +29,47 @@ const getSurfaceElevationValue = (elevation: SurfaceProps['elevation']) => {
   }
 };
 
+// ----------------------------------------------------------------------------
+// Surface.Header component
+
+export type SurfaceHeaderProps = React.ComponentPropsWithRef<'div'>;
+
+const SurfaceHeader = React.forwardRef<HTMLDivElement, SurfaceHeaderProps>(
+  (props, ref) => {
+    const { children, className, ...rest } = props;
+    return (
+      <div className={cx('iui-surface-header', className)} ref={ref} {...rest}>
+        {children}
+      </div>
+    );
+  },
+);
+
+// ----------------------------------------------------------------------------
+// Surface.Body component
+
+export type SurfaceBodyProps = React.ComponentPropsWithRef<'div'>;
+
+const SurfaceBody = React.forwardRef<HTMLDivElement, SurfaceBodyProps>(
+  (props, ref) => {
+    const { children, className, ...rest } = props;
+    return (
+      <div className={cx('iui-surface-body', className)} ref={ref} {...rest}>
+        {children}
+      </div>
+    );
+  },
+);
+
 export type SurfaceProps = {
+  /**
+   * Places children vertically when set to true
+   */
+  useLayout?: boolean;
+  /**
+   * Gives padding to the surface body
+   */
+  isPadded?: boolean;
```

Comment:
in itwinui-layouts we call it `padded`. lets make it consistent?
https://github.com/iTwin/iTwinUI-layouts/blob/63cb1a7440e5436a516b3456d31ba1a2e3d7c5a2/packages/layouts-react/src/components/PageLayout/Content.tsx#L18

Reply:
We use "is[Property]" in a lot of components.
Tile: isNew, isDisabled, isLoading
Header: isSlim
Text: isMuted, isSkeleton